@AH-64D Apache Longbow/Addons/fza_ah64_ihadss/functions/fn_controller.sqf

```diff
@@ -20,7 +20,7 @@ Author:
 ---------------------------------------------------------------------------- */
 params ["_heli"];
 
-if (!(vehicle player isKindOf "fza_ah64base") || !(alive _heli) || !(alive player) || !(isNull curatorCamera)) exitWith {
+if (!(vehicle player isKindOf "fza_ah64base") || !(alive _heli) || !(alive player) || !(isNull curatorCamera) || is3DEN || !(cameraView in ["INTERNAL", "GUNNER"]) || (isRemoteControlling player && !(getConnectedUAV player isKindOf "fza_ah64base"))) exitWith {
```

Comment:
This is starting to get a bit long for a single line, but not sure the best way to simplify

Reply:
readability 5/10
functionality 10/10